docs/upstream-differences.md

```diff
@@ -88,3 +88,70 @@ Once installer fork is ready:
 1. After this point, you should be able to create a dev cluster using the RP and it should use the new release.
 1. `make discoverycache`.
     * This command requires a running cluster with the new version.
+
+
+# Namespaces that ARO-RP uses mainly
```

Comment:
Always like to see a PR that comes with doc updates :+1:

Reply:
> Always like to see a PR that comes with doc updates +1

Hi Ross, Thanks for your time to review and great suggestions. Since Mikaila created the work item and should know more context about that, I just created this draft PR to let him review first to see if it's meet their needs.